site.yml

```diff
@@ -78,6 +78,36 @@
     - role: percona-backup
       tags: ['infra', 'percona']
 
+- name: install mongodb on primary
+  gather_facts: force
+  hosts: mongo_db[0]
+  vars:
+    mongodb_primary: True
+  roles:
+    - role: mongodb
+      tags: ['mongodb', 'mongodb-primary']
+      when: ceilometer.enabled|bool
+
+- name: install mongodb on secondaries
+  gather_facts: force
+  hosts: mongo_db:!mongo_db[0]
+  vars:
+    mongodb_primary: False
+  roles:
+    - role: mongodb
+      tags: ['mongodb', 'mongodb-secondary']
+      when: ceilometer.enabled|bool
```

Comment:
We currently do it like this to ensure that the replica set and db/users are instantiated first, then the secondaries and arbiter are run afterwards. if it was all one play, then the tasks would run in parallel and error